test/sandbox/test/integration_test.rb

```diff
@@ -214,7 +214,7 @@ def test_optional_rendering_component_depending_on_request_context
     assert_response :success
     assert_includes response.body, "Rendered"
 
-    cookies[:shown] = true
```

Comment:
Wait, isn't "shown" the opposite of "hide"? It would make sense if you were changing `shown == false` to `hide == true`, but...

Reply:
In this case, it's not the opposite, which I too found confusing. It's because it's the _past_ tense of `show`. That's why I changed this to `hide` for clarity.